### DISCUSSION TITLE
Can you put deeply-nested screens in the same stack?

### DISCUSSION THREAD

**Discussion by nandorojo** [expo/router#66] [Q&A]
Say I have the following layout:

- `(tabs)`
  - `index.tsx`
  - `users.tsx` (exports a stack)
  - `users/`
    - `index.tsx`
    - `[id]`
      - `edit.tsx`
      - `index.tsx`

## What I want

- Home tab
- Users tab
  - Stack with each of the following screens:
    - `[id]`
    - `[id]/edit`
    - `index`

## What happens

It seems like I need to make a nested stack for each folder. Specifically, I can't seem to get `[id]/edit` in the `users` stack somehow.

So I end up having to do this:

- `(tabs)`
  - `index.tsx`
  - `users.tsx` (exports a stack)
  - `users/`
    - `index.tsx`
    - `[id].tsx` (exports _another_ stack)
    - `[id]`
      - `edit.tsx`
      - `index.tsx`

I think this is a very common use case: wanting a single stack inside of a given tab.

**Comment by frankcalise** [comment]
Yeah I'm having this issue as well. The tab renders the initial route of the stack, but navigating to another screen ends up with an unmatched route.
